### ISSUE TITLE
Add "Main posts <=> Include replies" toggle to the users' feeds

### ISSUE THREAD

**Issue by dtonon** [mikedilger/gossip#493]
As per object.

**Comment by mikedilger** [comment]
Feeds will get a rework once we have lists.  I want user-defined feeds saved under user-created labels that become their menu choices under a "Feeds >" main menu.

**Comment by dtonon** [comment]
Maybe I was not clear: I mean to add the possibility, like in the main feed, to show only "Main posts" when I view the feed of a single user ("view their posts").

**Comment by mikedilger** [comment]
No, you were clear. That requires reworking the feeds. There is no feed enum for that possibility.
